### ISSUE TITLE
display:block style is not added when using jQuery 3.1

### ISSUE THREAD

**Issue by jeffblake** [Eonasdan/bootstrap-datetimepicker#1765]
When using the latest version of jQuery, I'm not getting any errors- however  the datepicker is not displaying. Inspecting the source, and swapping between jquery 2.x and 3.1, I noticed the only difference is that the `bootstrap-datetimepicker-widget` is not receiving an inline style of `display:block` on 3.1

Simple hack for the time being:

```
/* jquery 3 hack fix */
.bootstrap-datetimepicker-widget {
  display: block;
}
```

**Comment by usethe4ce** [comment]
I ended up having to do something a little different:

**Comment by usethe4ce** [comment]
I ended up having to do something a little different: `div.datepicker > div {
	display: block;
}`

**Comment by William-H-M** [comment]
Closed in  34bc7cc
